### PR TITLE
GET-1009 jmeter performance tests

### PR DESCRIPTION
**Pull request by bkarakashev**
### Context
JMeter-performance-tests

### CODE REVIEW

File: azure/jmeter/test.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	-Jhost=$host \
+	-n -t /$testFile -l $T_DIR/test-plan.jtl -j $T_DIR/jmeter.log \
+	-e -o $R_DIR
+
+# echo "==== jmeter.log ===="
+# cat $T_DIR/jmeter.log
+
+# echo "==== Raw Test Report ===="
+# cat $T_DIR/test-plan.jtl
+
```
Comment (tavy88) [inline azure/jmeter/test.sh]: Do we still need these ☝️ ?

Comment (bkarakashev) [reply azure/jmeter/test.sh]: nope I will clean it up

File: azure-pipelines-jmeter-performance-tests.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  displayName: Run JMeter tests
+  steps:
+  - task: Bash@3
+    displayName: Execute JMeter tests
+    inputs:
+      targetType: filePath
+      filePath: 'azure/jmeter/test.sh'
+      arguments: '$PWD test/jmeter-performance-tests.jmx $(host)'
+      workingDirectory: azure/jmeter
+      failOnStderr: false
```
Comment (cwrw) [inline azure-pipelines-jmeter-performance-tests.yml]: do we not want to fail this if it errors?

Comment (bkarakashev) [reply azure-pipelines-jmeter-performance-tests.yml]: It is a bit odd. as it is failing even if it is all pass. I will try to investigate and see if I can find out why is failing if there is no error... I assume is reading in the report where it says error % and its things that is an error... and marks the build as failed.

File: azure/jmeter/run.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+#
+# Run JMeter Docker image with options
+
+NAME="jmetertest"
+IMAGE="justb4/jmeter:latest"
+ROOTPATH=$1
+
+echo "$ROOTPATH"
+# Finally run
+docker stop $NAME > /dev/null 2>&1
```
Comment (cwrw) [inline azure/jmeter/run.sh]: discussed in person: attempt to remove

File: azure-pipelines-jmeter-performance-tests.yml (hunk 2 above)
Comment (cwrw) [reply azure-pipelines-jmeter-performance-tests.yml]: ignore for now, but docker fix might work, not a blocker

**Pull request merged**